cirq-core/cirq/protocols/resolve_parameters.py

```diff
@@ -67,7 +67,7 @@ def is_parameterized(val: Any) -> bool:
 
     A value is parameterized when it has an `_is_parameterized_` method and
     that method returns a truthy value, or if the value is an instance of
-    sympy.Basic.
+    sympy.Basic. Note this covers sympy constants such as `sympy.pi`.
```

Comment:
Tiny nit that can be ignored: I would perhaps change that line to

```
`sympy.Basic`. Note this covers SymPy constants such as `sympy.pi`.
```

but really, looking at the rest of the file, it has other places where capitalization and backticks are inconsistent, so it could be left to some later sweep to deal with such nonessential things.